docs/general/builtins/scalar-functions.rst

```diff
@@ -2043,7 +2043,7 @@ Returns: ``double precision``
 Returns Euler's number ``e`` raised to the power of the given numeric value.
 The output will be cast to the given input type and thus may loose precision.
 
-Returns: Same as input type.
+Returns: ``double precision``
```

Comment:
Looks like it was a documented behaviour making this a breaking change

Reply:
Interesting, I wonder about the rationale of this limitation

Reply:
Given that this was documented behavior this shouldn't go into 5.6.3, but instead it can be a regular change for 5.7